<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>479</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>833</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>JDKSAvdeccLogger</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel { 
	qproperty-alignment: AlignCenter; 
	font: &quot;Sans&quot;;
}

#labelTitle {
	font: 18pt;
}

#commonLogText {
	font: 13pt &quot;Courier&quot;;
}

#lineEditEntityID {
	font: 13pt &quot;Courier&quot;;
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>621</width>
        <height>61</height>
       </size>
      </property>
      <widget class="QCheckBox" name="checkBoxADP">
       <property name="geometry">
        <rect>
         <x>177</x>
         <y>5</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ADP</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxAECP">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>5</y>
         <width>58</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>AECP</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxACMP">
       <property name="geometry">
        <rect>
         <x>177</x>
         <y>32</y>
         <width>62</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ACMP</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditEntityID">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>4</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>191</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxEntityID">
       <property name="geometry">
        <rect>
         <x>326</x>
         <y>5</y>
         <width>83</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Entity ID:</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxJDKSLOG">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>32</y>
         <width>85</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>JDKS LOG</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonEnable">
       <property name="geometry">
        <rect>
         <x>87</x>
         <y>12</y>
         <width>84</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Enable</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>12</y>
         <width>75</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="commonLogText">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>91</height>
       </size>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Courier'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'.Lucida Grande UI';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelTitle">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>IEEE Std 1722.1-2013 (AVDECC) Logger v1.1 - https://github.com/jdkoftinoff/JDKSAvdeccLogger</string>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>833</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButtonEnable</tabstop>
  <tabstop>checkBoxADP</tabstop>
  <tabstop>checkBoxACMP</tabstop>
  <tabstop>checkBoxAECP</tabstop>
  <tabstop>checkBoxJDKSLOG</tabstop>
  <tabstop>checkBoxEntityID</tabstop>
  <tabstop>lineEditEntityID</tabstop>
  <tabstop>commonLogText</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>settingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>12</x>
     <y>467</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxADP</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>settingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxACMP</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>settingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAECP</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>settingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxJDKSLOG</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>settingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxEntityID</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>settingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditEntityID</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>settingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>461</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>commonLogText</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>settingsChanged()</slot>
 </slots>
</ui>
